### PR TITLE
Removing now unneeded package resolutions.

### PR DESCRIPTION
**Pull request by dennisoelkers**
## Motivation and Context

This change is removing package version resolutions which have become obsolete due to other package updates.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
     "core-js": "3",
     "crossfilter": "1.3.x",
     "dompurify": "^2.0.11",
+    "glob-parent": "6.0.2",
```
Comment (linuspahl) [inline graylog2-web-interface/package.json:63]: Just because I am curious, why are we now listing `glob-parent` as a dependency?

Comment (dennisoelkers) [reply graylog2-web-interface/package.json:63]: That is actually a mistake. It should actually be bumped in the lockfile only, I missed this in `git diff`. Thanks!

Comment (linuspahl) [reply graylog2-web-interface/package.json:63]: Alright!

**Comment by dennisoelkers** [comment]
Outdated.

**Pull request closed without merging**